mgmtd/mgmt_txn.c

```diff
@@ -2461,6 +2460,7 @@ int mgmt_txn_notify_be_adapter_conn(struct mgmt_be_client_adapter *adapter,
 		 * has failed.
 		 */
 		FOREACH_TXN_IN_LIST (mgmt_txn_mm, txn) {
+			/* XXX why is this config only? */
```

Comment:
Is this a TODO, FIXME ? please tag accordingly to make it easier track later.

Reply:
changed to: ```
			/* TODO: update with operational state when that is
			 * completed */
```